rfc/rfc023-x509credential.md

```diff
@@ -0,0 +1,556 @@
+internet. This RFC describes how x509 certificates can be used in the Nuts network to establish trust between parties by
+being able to link the x509 certificate to a Nuts identity by as a Verifiable Credential that is issued by the holder of
+the x509 identity.
+
+This RFC specifies the requirements and validation process for the `NutsX509Credential`, a W3C Verifiable Credential (
+VC) type issued by the subject of a x509 certificate, represented by a `did:x509` DID. The `NutsX509Credential` ensures
+strong alignment with the properties of the associated X.509 certificate and defines mechanisms to validate the
+credential and verify its association with a `did:x509` DID.
+
+## Status of this document
+
+This document is currently in draft. Feedback is welcome to improve the interoperability and robustness of the
+specification.
+
+### Copyright Notice
+
+![](../.gitbook/assets/license.png)
+
+## Introduction
+
+TThe [did:x509](https://trustoverip.github.io/tswg-did-x509-method-specification/) method aims to achieve interoperability between existing X.509 solutions and Decentralized Identifiers (DIDs) to support operational models in which a full transition to DIDs is not achievable or desired yet. It supports X.509-only verifiers as well as DID-based verifiers supporting this DID method.
+
+The `NutsX509Credential` is a W3C Verifiable Credential type designed for use cases where trust anchors are based on X.509
+certificates. It leverages the `did:x509` method, as specified in
+the [Trust Over IP DID:X509 Method Specification](https://trustoverip.github.io/tswg-did-x509-method-specification/).
+
+By aligning credential subject validation with the fields of the associated `did:x509` DID and enforcing
+certificate revocation checks, the `NutsX509Credential` ensures integrity and adherence to the PKI trust model.
+
+
+## Definitions
+
+- **NutsX509Credential**: A Verifiable Credential whose issuer is a `did:x509` DID and whose structure adheres to the
+  requirements in this document.
+- **did:x509**: A Decentralized Identifier (DID) method specified by the Trust Over IP Foundation, where the DID is
+  derived from an X.509 certificate.
+- **Issuer Certificate**: The X.509 certificate associated with the `did:x509` DID that issued the credential.
+- **Credential Subject**: The entity described by the credential.
+- **Revocation Check**: The process of verifying the revocation status of the issuer certificate using mechanisms like
+  OCSP or CRL.
+
+## Used standards and technologies
+
+This RFC builds on the following standards and technologies:
+
+* [X.509 Certificate Standard](https://datatracker.ietf.org/doc/html/rfc5280)
+* [JSON Web Signature (JWS)](https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.6)
+* [did:x509 method specification](https://trustoverip.github.io/tswg-did-x509-method-specification/), with modifications
+* [Verifiable Credentials Data Model v1.1](https://www.w3.org/TR/vc-data-model/)
```

Comment:
Is this version correct?

Reply:
Shouldn't it be: https://www.w3.org/TR/vc-data-model-2.0/

Reply:
we use 1.1, 2.0 is still in draft